README.adoc

```diff
@@ -201,37 +229,42 @@ You can build everything from the root of the repository.
 
 `./mvnw clean install`
 
-But, this may not be what you are interested in doing since you are probably interested in a single application or a few of them.
-In order to build the functions and applications that you are interested in, you need to build them selectively as shown below.
+However, this may not be what you are interested in since you are probably interested in a single application or a few of them.
+
+To build the functions and applications that you are interested in, you need to build them selectively, as shown below.
 
-==== Building functions
+==== Building Functions
 
 `./mvnw clean install -f functions`
 
 You can also build a single function or group of functions.
-For e.g if you are only interested in jdbc-supplier and log-consumer, do the following.
+For example, if you are only interested in `jdbc-supplier` and `log-consumer`, do the following.
 
 `./mvnw clean install -pl :jdbc-suppler,:log-consumer`
 
-==== Building core for Stream Applications
+==== Building Stream Applications Core
+
+If you want to re-run the common core build, you can build it with the following.
 
 `./mvnw clean install -f applications/stream-applications-core`
 
-=== Building the applications
+=== Building Stream applications
+
+Let's assume that you want to build a `jdbc-source` application based on Kafka Binder in Spring Cloud Stream and Log Sink
+application based on Rabbit binder.
 
-Let's assume that you want to build JDBC Source application based on Kafka Binder in Spring Cloud Stream and Log Sink application based on Rabbit binder.
 Here is what you need to do.
-Assuming that you built both functions and stream-applications-core as above.
+Assuming that you already built both functions and stream-applications-core as above,
 
 ```
 ./mvnw clean package -pl :jdbc-source
 cd applications/source/jdbc-source/apps/jdbc-source-kafka
 ./mvnw clean package
 ```
 
-This will generate the Kafka binder based uber jar in the target folder.
+This will generate the Kafka binder based uber-jar in the target folder.
```

Comment:
Doesn't look like an "uber-jar" is a thing: https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-executable-jar-format.html#executable-jar-nested-jars. So, previous sentence was correct.

Reply:
No hyphen here.